docs/ocis/config.md

```diff
@@ -0,0 +1,165 @@
+---
+title: "Configuration"
+date: "2021-11-09T00:03:16+0100"
+weight: 2
+geekdocRepo: https://github.com/owncloud/ocis
+geekdocEditPath: edit/master/ocis/templates
+geekdocFilePath: config.md
+---
+
+{{< toc >}}
+
+## Configuration Framework
+
+In order to simplify deployments and development the configuration model from oCIS aims to be simple yet flexible.
+
+## Overview of the approach
+
+{{< svg src="ocis/static/ocis-config-redesign.drawio.svg" >}}
+
+## In-depth configuration
+
+Since we include a set of predefined extensions within the single binary, configuring an extension can be done in a variety of ways. Since we work with complex types, having as many cli per config value scales poorly, so we limited the options to config files and environment variables, leaving cli flags for shared values, such as config file sources (`--config-file`) or logging (`--log-level`, `--log-pretty`, `--log-file` or `--log-color`).
```

Comment:
is there a `--config-file` option? `ocis ... help` doesn't display any

Reply:
no. You're right. It makes sense removing it from the docs while is not yet there 👍